geoh5py/io/h5_reader.py

```diff
@@ -230,6 +230,14 @@ def fetch_concatenated_attributes(
 
                     return json.loads(as_str_if_utf8_bytes(attribute))
 
+                if label == "Attributes Jsons":
+                    attribute = group[label][()]
```

Comment:
what does [()] do?

Reply:
That's an h5py syntax to get an array out